From 95388b0797459d10b30d0f3b00d9dc1ef8951f30 Mon Sep 17 00:00:00 2001
From: WangJing <wangjing@mprc.pku.edu.cn>
Date: Mon, 10 Oct 2011 18:06:33 +0800
Subject: [PATCH 012/641] UniCore64:Add file pci.h in Kbuild

Fixed error:
In file included from drivers/hwmon/hwmon.c:23:0:
include/linux/pci.h:1307:21: fatal error: asm/pci.h: No such file or directory
In file included from lib/devres.c:1:0:
include/linux/pci.h:1307:21: fatal error: asm/pci.h: No such file or directory

Signed-off-by: WangJing <wangjing@mprc.pku.edu.cn>
---
 arch/unicore64/include/asm/Kbuild |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/unicore64/include/asm/Kbuild b/arch/unicore64/include/asm/Kbuild
index fdba13e..9126c2a 100644
--- a/arch/unicore64/include/asm/Kbuild
+++ b/arch/unicore64/include/asm/Kbuild
@@ -64,3 +64,4 @@ generic-y += types.h
 generic-y += uaccess.h
 generic-y += unaligned.h
 generic-y += user.h
+generic-y += pci.h
-- 
1.7.9.5

